### PR TITLE
Move ResizeObserver to iframes

### PR DESCRIPTION
**Pull request by Joozty**
I slightly refactored to implementation. The workflow is following:

1. Load iframe with `elementId` parameter
2. If that parameter exists and the page is loaded from within the iframe and ResizeObserver API exist
     - Listen on any resize change and update height on `elementId` element from parent
3. Else the arbitrary height is applied and user sees scrollbar (hope it never happens)

### CODE REVIEW

File: public/secure/login.html [hunk 1]
```diff
@@ -2,7 +2,6 @@
 <html lang="en">
 
 <meta charset="UTF-8">
-<meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (viktor-yakubiv) [inline public/secure/login.html:5]: I see this kept in other places. Leftover?

File: public/secure/resize.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+function handleResizeEvent(entries, elementId) {
+  // https://stackoverflow.com/a/58701523/5594539
+  window.requestAnimationFrame(() => {
+    if (!Array.isArray(entries) || !entries.length) return
+
+    const iframe = window.parent.document.getElementById(elementId)
+    iframe.style.height = `${window.document.body.offsetHeight}px`
```
Comment (viktor-yakubiv) [inline public/secure/resize.js:7]: Really? You can do so? It's even better 👏

By the way, did you check if some iframe attribute could deny this?

**Review by viktor-yakubiv — APPROVED** [review]
Great job 👍

File: public/secure/login.html (hunk 1 above)
Comment (Joozty) [reply public/secure/login.html:5]: I was suspecting this was causing the loop exceed but I was wrong and forgot to add it back. Thanks

File: public/secure/resize.js (hunk 2 above)
Comment (Joozty) [reply public/secure/resize.js:7]: I think you can do so only if you are on the same origin.

**Pull request merged**